pkg/prober/intrumentation.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 const (
-	ready   = "ready"
-	healthy = "healthy"
+	ready    = "ready"
+	notReady = "not-ready"
```

Comment:
Do we need notReady status? !ready just means notReady, right?

Reply:
Maybe we can leave `notReady` and use it in the log calls? So, we'd have:
```go
		level.Warn(p.logger).Log("msg", "changing probe status", "status", notReady, "reason", err)
```